CLIENT-SPECIFICATION.md

```diff
@@ -31,23 +31,23 @@ This section describes the standardised command-line interface (CLI) for clients
 
 ### Arguments
 
-A number of command-line options MUST be supported (unless otherwise specified) if a CLI is implemented:
+Several command-line options MUST be supported (unless otherwise specified) if a CLI is implemented:
```

Comment:
Let's make this explicit:


```suggestion
The following command-line options MUST be supported (unless otherwise specified) if a CLI is implemented:
```